### PR TITLE
add navigation and Router

### PR DESCRIPTION
**Pull request by kamila-wilczynska**
Structure of folders slightly changed.
Navigation component added. Navigation styles implemented.
Added Home and About file.
Outlet created in new file called Main

image

Root file created where the structure of website have been moved
image

React Router implemented in App.jsx
image

image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,20 +1,38 @@
 import './assets/css/App.css';
-import Header from "./components/Header/Header";
-import SiteNav from './components/SiteNav/SiteNav';
+// import Header from "./components/Header/Header";
+// import SiteNav from './components/SiteNav/SiteNav';
 import MainContent from './components/MainContent/MainContent';
-import Footer from './components/Footer/Footer';
+// import Footer from './components/Footer/Footer';
```
Comment (orangespaceman) [inline src/App.jsx]: If these files are no longer used you could remove them here

File: src/components/About/About.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export default function About() {
+  return (
+    <div>
+      <p>About</p>
+    </div>
```
Comment (orangespaceman) [inline src/components/About/About.jsx]: This file has a `<div>` around the content but it's just a `<>` in Home - this is probably fine but might be worth making it consistent

File: src/components/Main/Main.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+function MainContent() {
+	return (
+		<div>
+			<Outlet />
+		</div>
+	);
+}
+
+export default MainContent;
```
Comment (orangespaceman) [inline src/components/Main/Main.jsx]: This file is called `Main` but the function is `MainContent` - also this is done in a slightly different way to Home and About - the end result is the same, it's just a little inconsistent

File: src/components/Root/Root.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// import { Outlet } from "react-router-dom";
+import Header from "../Header/Header";
+import SiteNav from "../SiteNav/SiteNav";
+// import MainContent from '../MainContent/MainContent';
```
Comment (orangespaceman) [inline src/components/Root/Root.jsx]: The two comments in this file can probably. be removed

**Review by orangespaceman — COMMENTED** [review]
Looks great! I've added a few small comments that you might want to tweak

File: src/components/Root/Root.jsx (hunk 4 above)
Comment (kamila-wilczynska) [reply src/components/Root/Root.jsx]: Thanks Pete, All comments removed!

**Review by BlueElyas — APPROVED** [review]
Looks good, I think we can ignore the netlify warning because that was before the newest commit. Otherwise if problems appear, we'll just solve them 😎

**Review by Sebbybobbler — APPROVED** [review]
Really informative PR! :D

**Review by sfbennett — APPROVED** [review]

**Pull request merged**